snapcraft.yaml

```diff
@@ -12,6 +12,144 @@ grade: devel
 assumes: [snapd2.55.5]
 
 parts:
+  chisel-libs:
+    plugin: nil
+    build-packages:
+      - golang
+    source: https://github.com/Meulengracht/chisel-releases.git
```

Comment:
How do you enable PPAs? We usually enable https://launchpad.net/~ucdev/+archive/ubuntu/base-ppa But for now on core24, it is not needed. But it probably will be. Is that in `chisel.yaml` there?

Reply:
I will owe you an answer on how this works. Since packages are fetched in a building context in snapcraft, I guess we simply add it the way you would normally do to enable PPAs. Snapcraft has built-in syntax for this?

Reply:
Do you mean that chisel takes the host apt configuration from `/etc/apt`?

Reply:
PPAs for chisel are not supported currently: https://github.com/canonical/chisel/issues/98

From the code it looks like the PPA url is hardcoded: https://github.com/canonical/chisel/blob/a88e10523947ae15e1f87116b56803e13d3324b4/internal/archive/archive.go#L129

I'll bring this up with the chisel team